coalib/bears/Bear.py

```diff
@@ -273,6 +277,10 @@ def __init__(self,
                               time limit, use 0.
         :raises TypeError:    Raised when ``message_queue`` is no queue.
         :raises RuntimeError: Raised when bear requirements are not fulfilled.
+        :param debugger       A flag which enable debugger on those bears which
+                              are specified in ``debug_bears`` settings.
```

Comment:
why do you introduce documentation about the debugger in the commit for the profiler?

Reply:
Yeah I accidentally squashed in my last update